### ISSUE TITLE
Trix editor needs @keydown.stop

### ISSUE THREAD

**Issue by ryanhungate** [64robots/nova-fields#55]
There is a bug with this editor when trying to add a question mark to the text area - and what's missing is the @keydown.stop reference to stop that from happening. 

```
<template>
    <trix-editor
        ref="theEditor"
        @keydown.stop
        @trix-change="handleChange"
        @trix-initialize="initialize"
        @trix-attachment-add="handleAddFile"
        @trix-attachment-remove="handleRemoveFile"
        @trix-file-accept="handleFileAccept"
        :value="value"
        :placeholder="placeholder"
        class="trix-content"
    />
</template>
```

**Comment by disitec** [comment]
This has been fixed in the upcoming release 0.6.2. Thanks for the tip!

**Comment by ryanhungate** [comment]
👍 thanks for the package too! I think this should most certainly be merged into the core of Nova...
